networkx/algorithms/centrality/domirank.py

```diff
@@ -0,0 +1,305 @@
+"""DomiRank Centrality: By Marcus Engsig (@mengsig)"""
+
+import networkx as nx
+from networkx.utils import not_implemented_for
+
+__all__ = ["domirank"]
+
+
+# @nx._dispatchable(edge_attrs="weight")
+@not_implemented_for("multigraph")
+def domirank(
+    G, analytical=False, sigma=0.95, dt=0.1, epsilon=1e-5, max_iter=1000, check_step=10
+):
+    r"""Compute the DomiRank centrality for the graph `G`.
+
+    DomiRank centrality [1]_ computes the centrality for a node by adding
+    1 minus the centrality of its neighborhood. This essentially finds the
+    dominance of a node in its neighborhood, where the parameter $\sigma$ determines
+    the amount of competition in the system. The competition parameter $\sigma$
+    tunes the balance of DomiRank centrality's integration of local and global topological
+    information, to find nodes that are either locally or globally dominant. It is
+    important to note that for the iterative formulation of DomiRank (as seen below),
+    the competition parameter is bounded: $\sigma \in [0,1/(-\lambda_N)]$.
```

Comment:
```suggestion
    the competition parameter is bounded: $\sigma \in [0,1/|\lambda_N|]$.
```

Do we have a guarantee that `\lambda_N` is negative?